core/test/oracle/Store.js

```diff
@@ -61,6 +61,14 @@ contract("Store", function(accounts) {
     let highFee = { rawValue: web3.utils.toWei("1", "ether") };
     assert(await didContractThrow(store.setFixedOracleFeePerSecond(highFee, { from: owner })));
 
+    // Can set weekly late fees to less than 100%.
+    await store.setWeeklyDelayFee({ rawValue: web3.utils.toWei("0.99", "ether") }, { from: owner });
+
+    // Disallow setting fees higher than 100%.
```

Comment:
nit: comment doesn't match impl -- you test == 100%, but the comment implies that > 100% is not allowed.